spec/components/schemas.yaml

```diff
@@ -575,15 +575,15 @@ components:
       example: komb
       title: Product variants
       type: string
-    geometryGeoJSON:
-      oneOf:
-        # - $ref: "./schemas.yaml#/components/schemas/pointGeoJSON"
-        # - $ref: "./schemas.yaml#/components/schemas/multipointGeoJSON"
-        # - $ref: "./schemas.yaml#/components/schemas/linestringGeoJSON"
-        # - $ref: "./schemas.yaml#/components/schemas/multilinestringGeoJSON"
-        - $ref: "./schemas.yaml#/components/schemas/polygonGeoJSON"
-        # - $ref: "./schemas.yaml#/components/schemas/multipolygonGeoJSON"
-        # - $ref: "./schemas.yaml#/components/schemas/geometrycollectionGeoJSON"
+    # geometryGeoJSON:
+    #   oneOf:
+    #     # - $ref: "./schemas.yaml#/components/schemas/pointGeoJSON"
+    #     # - $ref: "./schemas.yaml#/components/schemas/multipointGeoJSON"
+    #     # - $ref: "./schemas.yaml#/components/schemas/linestringGeoJSON"
+    #     # - $ref: "./schemas.yaml#/components/schemas/multilinestringGeoJSON"
+    #     - $ref: "./schemas.yaml#/components/schemas/polygonGeoJSON"
+    #     # - $ref: "./schemas.yaml#/components/schemas/multipolygonGeoJSON"
+    #     # - $ref: "./schemas.yaml#/components/schemas/geometrycollectionGeoJSON"
```

Comment:
I was not exactly sure, why there was only polygonGeoJSON available here and the rest was commented out.
In the currently productive specs for item's PATCH all geometry types are described. For all other methods, only `polygonGeoJSON` is available.
I supposed that all options should be valid for any method and implemented it that way in this PR.
So I'll delete this commented out section, once this PR is approved.